connector/src/main/java/org/geysermc/connector/network/session/auth/AuthData.java

```diff
@@ -1,12 +1,14 @@
 package org.geysermc.connector.network.session.auth;
 
+import java.util.UUID;
+
 import lombok.AllArgsConstructor;
 import lombok.Getter;
-
-import java.util.UUID;
+import lombok.ToString;
 
 @Getter
 @AllArgsConstructor
+@ToString
```

Comment:
`@ToString` can be kept for debugging purposes but the rest of this can be reverted.